dashboard/modules/job/cli.py

```diff
@@ -310,3 +311,25 @@ def job_logs(address: Optional[str], job_id: str, follow: bool):
             )
     else:
         print(client.get_job_logs(job_id), end="")
+
+
+@job_cli_group.command("list", help="List all jobs together with their info.")
+@click.option(
+    "--address",
+    type=str,
+    default=None,
+    required=False,
+    help=(
+        "Address of the Ray cluster to connect to. Can also be specified "
+        "using the RAY_ADDRESS environment variable."
+    ),
+)
+@add_click_logging_options
+def list_jobs(address: Optional[str]):
+    """Lists all running jobs and their information.
+
+    Example:
+        >>> ray job list
+    """
+    client = _get_sdk_client(address)
+    print(pprint.pformat(client.list_jobs()))
```

Comment:
please use the Ray CLI logger to make sure the output is standardized across calls